.github/workflows/pr-preview.yml

```diff
@@ -0,0 +1,34 @@
+name: PR-Preview
+
+on:
+  pull_request:
+    branches: [ main ]
+
+env:
+  REACT_APP_NETWORK_IDS: "100,10200"
+  GITHUB_ENV: "./"
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: 'package.json'
```

Comment:
This doesn't seem to be documented really well but is seemingly supported indeed. A few references in case we need them at some point:
- https://github.com/actions/setup-node/pull/553/files
- https://github.com/actions/setup-node/pull/485